api/all/src/main/java/io/opentelemetry/api/trace/Span.java

```diff
@@ -162,6 +163,26 @@ default Span setAttribute(AttributeKey<Long> key, int value) {
     return setAttribute(key, (long) value);
   }
 
+  /**
+   * Sets attributes to the {@code Span}. If the {@code Span} previously contained a mapping for any
+   * of the keys, the old values are replaced by the specified values.
+   *
+   * @param attributes the attributes
+   * @return this.
+   */
+  default Span setAttributes(Attributes attributes) {
```

Comment:
This seems more like `addAll` to me than `set`. The latter implies that new attributes completely replace the old ones.

Reply:
I'd almost argue that `set*` should be called `put*` given the `Map`-like semantics.  But I'll let you guys argue/settle on the names.  😁 

I've worked out my Gradle/IDE issues and hopefully will get some unit tests up later this morning.  Apologies for the rapid fire commits last night if they resulted in notifications.

Reply:
`setAll` is another option for this.

Reply:
Map-like would be then `putAll` :)

Reply:
Yep.  I'm game with whatever name you guys prefer, let me know and I'll update the PR.

Reply:
well, IMO, all the rest of the methods, for good or for ill, are `set*`, so we should definitely start with `set`. :)